package.json

```diff
@@ -2,21 +2,22 @@
     "name": "posthog-plugin-server",
     "version": "0.2.0",
     "description": "PostHog Plugin Server",
-    "types": "dist/index.d.ts",
-    "main": "dist/index.js",
+    "types": "dist/src/index.d.ts",
+    "main": "dist/src/index.js",
     "scripts": {
         "test": "jest",
         "start": "yarn start:dev",
-        "start:dist": "node dist/index.js --config \"{\\\"BASE_DIR\\\": \\\"../posthog\\\"}\"",
+        "start:dist": "node dist/src/index.js --config \"{\\\"BASE_DIR\\\": \\\"../posthog\\\"}\"",
         "start:dev": "ts-node-dev --exit-child src/index.ts --config \"{\\\"BASE_DIR\\\": \\\"../posthog\\\"}\"",
         "build": "yarn clean && yarn compile",
         "clean": "rimraf dist/*",
-        "compile": "rollup -c",
+        "compile": "tsc -p .",
         "lint": "eslint .",
         "lint:fix": "eslint --fix .",
         "prettier": "prettier --write .",
         "prettier:check": "prettier --check .",
-        "prepublish": "yarn test && yarn build"
+        "prepublishOnly": "yarn test && yarn build",
```

Comment:
Looks a bit out of place with camelCase, BTW I think we could do away with "only"?

Reply:
That's unfortunately `npm`/`yarn` requiring this capitalisation.

- `prepublish` runs the scripts at both "yarn install" and "yarn publish"
- `prepublishOnly` runs the script at only "yarn publish"

I don't want all the tests to run at `yarn` all the time. Otherwise I get too many errors and it just seems weird. Some tests also a working database, which is created with the `tasks:db-init` script. I'm somewhat hesitant to add this to the test script. Also, [posthog-js also uses `prepublishOnly`](https://github.com/PostHog/posthog-js/blob/master/package.json#L15)

Reply:
Oh, I had no idea that that was something `prepublish` is doing and `prepublishOnly` isn't. Good to know.